### PR TITLE
Feature/data flag

### PR DESCRIPTION
**Pull request by imbenwolf**

### CODE REVIEW

**Comment by Exef** [comment]
@georgercarder please review this post merge in your time.

**Review by georgercarder — COMMENTED** [review]
Code changes look good, but will approve only after I am able to clone, build, test locally. See internal thread where I'm asking details of build. Thanks in advance.

File: test/VM.js [hunk 1]
```diff
@@ -71,6 +79,55 @@ describe("VM", function () {
     console.log(`Msg.sender: ${receipt.gasUsed.toNumber()} gas`);
   });
 
+  it("Should call fallback", async () => {
+    const commands = [[fallback, "", "0x2180ffffffffff", "0xff"]];
+    const state = ["0x"];
+
+    const tx = await execute(commands, state);
+    await expect(tx).to.not.emit(fallbackContract, "LogBytes");
```
Comment (georgercarder) [inline test/VM.js:87]: why is this "to NOT emit"?

Comment (georgercarder) [reply test/VM.js:87]: Answered in discord: "since no msg.data is supplied".

**Pull request merged**